src/test/test_options.c

```diff
@@ -422,11 +422,11 @@ get_options_test_data(const char *conf)
   config_line_t *cl=NULL;
   options_test_data_t *result = tor_malloc(sizeof(options_test_data_t));
   result->opt = options_new();
-  result->old_opt = options_new();
+  result->old_opt = NULL;
   result->def_opt = options_new();
 
   options_init(result->opt);
-  options_init(result->old_opt);
+  // options_init(result->old_opt);
```

Comment:
Should we just delete this line?
I don't quite understand why it was disabled, so maybe we need a comment?